From 2778018fd844e20464bdabdedaac5edd2b0c91c1 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 6 May 2020 15:33:26 +0800
Subject: [PATCH] rk: arm: do not build Image.gz/lz4 when build img

Sometimes Image is not ready when build Image.gz. Do not know why.

Fixes: 6bec2128bb01 ("rk: arm: support build Image.gz and Image.lz4")
Change-Id: Iafa7a7cd138d7169f5f198977be0c9d01765a5fc
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/Makefile b/arch/arm/Makefile
index f93a91ad8270..790d78596faa 100644
--- a/arch/arm/Makefile
+++ b/arch/arm/Makefile
@@ -406,7 +406,7 @@ LOGO := $(notdir $(wildcard $(srctree)/logo.bmp))
 LOGO_KERNEL := $(notdir $(wildcard $(srctree)/logo_kernel.bmp))
 endif
 
-%.img: %.dtb kernel.img Image.gz Image.lz4 $(LOGO) $(LOGO_KERNEL)
+%.img: %.dtb kernel.img $(LOGO) $(LOGO_KERNEL)
 	$(Q)$(srctree)/scripts/mkimg --dtb $*.dtb
 
 CLEAN_DIRS += out
-- 
2.35.3

